src/components/NarrowScreenTemplate.tsx

```diff
@@ -65,25 +86,25 @@ const styles = StyleSheet.create({
     backgroundColor: palette.modalBackgroundOverlay,
   },
   container: {
-    flex: 1,
-    width: 528,
     ...getElevation(4),
+    width: 528,
```

Comment:
I know you just moved this line and probably I'm the author but I guess it would be worth to have this `528` as named constant ;)